Cargo.toml

```diff
@@ -29,13 +30,13 @@ opt-level = 3
 
 [dependencies]
 byteorder = "1"
-crossbeam-channel = "0.2.4"
+crossbeam-channel = "0.3"
 csv = "1"
-csv-index = "0.1.5"
+csv-index = "0.1"
```

Comment:
This should probably remain as `0.1.5` (unless you've specifically tested that `xsv` can work with `0.1.0`).

Reply:
I was just going off of semver that it should be fine... but I can specify the exact version if you want.  I didn't do any special testing or anything.